### PR TITLE
Clarify how infections are estimated

### LINKED ISSUE #522: Clarify README wording
This is a bit confusing I think as it seems to suggest infections are first imputed then mapped forward (i.e. the original EpiNow approach). Would be good to clarify that infections are a latent variable in the model.

_Originally posted by @sbfnk in https://github.com/epiforecasts/EpiNow2/pull/521#discussion_r1406517131_

### PR DESCRIPTION
**Pull request by jamesmbaazam**
## Description

This PR closes #522.

The process of estimating infections was previously described in a way to suggest that it was imputed, but has now been clarified to say that it is estimated as a latent variable.

## Initial submission checklist

- [x] My PR is based on a package issue and I have explicitly linked it.
- [x] I have tested my changes locally (using `devtools::test()` and `devtools::check()`).
- [ ] I have added or updated unit tests where necessary.
- [x] I have updated the documentation if required and rebuilt docs if yes (using `devtools::document()`).
- [x] I have followed the established coding standards (and checked using `lintr::lint_package()`).
- [ ] I have added a news item linked to this PR.

## After the initial Pull Request 

- [ ] I have reviewed Checks for this PR and addressed any issues as far as I am able.

### CODE REVIEW

**Comment by jamesmbaazam** [comment]
Not sure if this is NEWs worthy.

File: README.Rmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ Forecasting is also supported for the time-varying reproduction number, infectio
 
 <details> <summary> More details </summary>
 
-`{EpiNow2}` estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). Imputed infections are then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
+`{EpiNow2}` estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). True infections treated as latent and unobserved, estimated, and then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity.
```
Comment (sbfnk) [inline README.Rmd:28]:
```suggestion
`{EpiNow2}` estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). True infections treated as latent and unobserved, are estimated and then mapped to observed data (for example cases by date of report) via one or more delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
```

**Review by sbfnk — CHANGES_REQUESTED** [review]
minor rephrasing suggeted

File: README.Rmd (hunk 1 above)
Comment (jamesmbaazam) [reply README.Rmd:28]: I actually wanted to say "True infections [ARE] treated as latent and unobserved, estimated, and ..."

What do you think?

**Review by sbfnk — APPROVED** [review]

**Pull request merged**